ci/docker/install/arm8_python.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+set -ex
+apt-get update || true
+
+# Avoid interactive package installers such as tzdata.
+export DEBIAN_FRONTEND=noninteractive
+
+apt-get -y install python-pip
+
+pip install --upgrade \
+                        pip \
+                        setuptools
+
+pip install \
+                        graphviz \
```

Comment:
@lebeg Looking at this again... I added an [issue about this](https://github.com/apache/incubator-mxnet/issues/14918) last week. I ask why we need both the system version of graphviz and the python version. It appears that they offer different features and ways to invoke, but it isn't clear to me what we need. I'm going with the assumption that `mxnet.viz` need one or both of these.
You can't pin a system version - you can do some scripting checks to look for packages and try to get the right one but there's no >= feature for apt from what I can tell. So that one is unpinned, but the pip one I can pin.

Reply:
Strange, but probably ok.